docs/core/tools/custom-templates.md

```diff
@@ -123,15 +123,15 @@ dotnet new -u <NUGET_PACKAGE_ID>
 
 ### To uninstall a template from a local nupkg file
 
-When you wish to uninstall the template, don't attempt to use the path to the *nupkg* file. *Attempting to uninstall a template using `dotnet new -u <PATH_TO_NUPKG_FILE>` fails.* Reference the package by its `id`:
+When you wish to uninstall the template, don't attempt to use the path to the *nupkg* file. Attempting to uninstall a template using `dotnet new -u <PATH_TO_NUPKG_FILE>` fails. Reference the package by its `id`:
```

Comment:
nit: This is somewhat flowery. Better is just, "To uninstall the template, don't use..."